### ISSUE TITLE
Publish latest version of uniextract on github

### ISSUE THREAD

**Issue by Tarek-Hasan** [Bioruebe/UniExtract2#355]
Last version of uniextract released on github is from 2020.
Please, publish latest version of uniextract on github instead of just giving update through update manager.

**Comment by zedd-0** [comment]
yea that will be nice...
i miss that, too.

**Comment by FabianLoRs** [comment]
+1

**Comment by AllanHigdon** [comment]
Instead of updating through the program itself, using the "Install beta updates" option, you can download the current development build.
https://update.bioruebe.com/uniextract/files/nightly.zip
The most recent is from 6/23/2023. It includes v2.0.0 RC4.

**Comment by 03302024** [comment]
Just a follow up for this feature request.

**Comment by raddyst** [comment]
> Just a follow up for this feature request.

https://github.com/Bioruebe/UniExtract2#nightly-builds

**Comment by 03302024** [comment]
The request is: "Publish latest version ... on github"
https://github.com/Bioruebe/UniExtract2/releases

**Comment by raddyst** [comment]
> The request is: "Publish latest version ... on github"

You see a release? Or full changelog of RC4?

**Comment by zedd-0** [comment]
> > The request is: "Publish latest version ... on github"
> 
> You see a release? Or full changelog of RC4?

No, the purpose of this issue is to remind the developer that release cycles are lagging behind the internal app updater.

**Comment by Wisewiz001** [comment]
The latest (RC4) is always available at Neowin:

https://www.neowin.net/software/universal-extractor-2-uniextract2-200-rc4/

**Comment by raddyst** [comment]
> No, the purpose of this issue is to remind the developer that release cycles are lagging behind the internal app updater.

Seriously? 

![image](https://github.com/Bioruebe/UniExtract2/assets/8584929/20f09053-4e2a-4c6e-a41d-0d33c3f00c65)

**Comment by zedd-0** [comment]
> > No, the purpose of this issue is to remind the developer that release cycles are lagging behind the internal app updater.
> 
> Seriously? https://github.com/Bioruebe/UniExtract2#nightly-builds
> 
> ![image](https://private-user-images.githubusercontent.com/8584929/318210921-20f09053-4e2a-4c6e-a41d-0d33c3f00c65.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE4MTcwNDEsIm5iZiI6MTcxMTgxNjc0MSwicGF0aCI6Ii84NTg0OTI5LzMxODIxMDkyMS0yMGYwOTA1My00ZTJhLTRjNmUtYTQxZC0wZDMzYzNmMDBjNjUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMzBUMTYzOTAxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZWFhZGI3ZjkxMWI4MzU1ZjU4NmY3Yzg2MGQ1ZTIxZmUwMTM1MzFlMTUyMjNiODAyNmZlODhkODM2ZWE5ZDExMSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.dAmJH6u8mKnJYRd4mjo7seP45oal7A_IgTz-vR3n7NA)

i cannot use the app on an non window or non x86 system.
that's why a online source is so important.

**Comment by raddyst** [comment]
> The latest (RC4) is always available at Neowin:

Link at neowin from bioruebe.com

**Comment by raddyst** [comment]
> that's why a online source is so important.

Try "Tiny AutoIt3 Decompiler"

**Comment by Korb** [comment]
![Image](https://github.com/user-attachments/assets/246a1a50-530a-4e5d-ab29-86298e3cb7e5)

Here's the method I would prefer to use for notifications when a new version of an app is available. In the case of the `Universal Extractor 2` aka `UniExtract2` app, this does not work.
